rust/sdk-processor/src/processors/fungible_asset_processor.rs

```diff
@@ -0,0 +1,151 @@
+use crate::{
+    config::{
+        db_config::DbConfig, indexer_processor_config::IndexerProcessorConfig,
+        processor_config::ProcessorConfig,
+    },
+    steps::{
+        common::latest_processed_version_tracker::LatestVersionProcessedTracker,
+        fungible_asset_processor::{
+            fungible_asset_extractor::FungibleAssetExtractor,
+            fungible_asset_storer::FungibleAssetStorer,
+        },
+    },
+    utils::{
+        chain_id::check_or_update_chain_id,
+        database::{new_db_pool, run_migrations, ArcDbPool},
+        starting_version::get_starting_version,
+    },
+};
+use ahash::AHashMap;
+use anyhow::Result;
+use aptos_indexer_processor_sdk::{
+    aptos_indexer_transaction_stream::{TransactionStream, TransactionStreamConfig},
+    builder::ProcessorBuilder,
+    common_steps::TransactionStreamStep,
+    traits::IntoRunnableStep,
+};
+use processor::worker::TableFlags;
+use serde::{Deserialize, Serialize};
+use tracing::{debug, info};
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct FungibleAssetProcessorConfig {
```

Comment:
Can this be abstracted out in someway? looks the same as the `EventsProcessorConfig`

Reply:
Abstracted out into DefaultProcessorConfig